src/main/resources/api.properties

```diff
@@ -0,0 +1,3 @@
+api.client=apache
```

Comment:
it is not secure to store credentials and other sensitive data in git repository. Usually The secret manager is used for that,